app/controllers/supervisor/comments_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Supervisor::CommentsController < Supervisor::BaseController
+
+  def create
+    comment = Comment.new(comment_params)
+
+    if comment.save
+      CommentMailer.email(comment).deliver_later
```

Comment:
Shouldn't we mail only if text is present? Maybe this could be an `after_save` on the model.